src/control/server/init/setup_spdk.sh

```diff
@@ -19,11 +19,12 @@ else
 
 	# build arglist manually to filter missing directories/files
 	# so we don't error on non-existent entities
-	for glob in /dev/hugepages '/dev/uio*'		\
+	for glob in '/dev/hugepages' '/dev/uio*'		\
 		'/sys/class/uio/uio*/device/config'	\
 		'/sys/class/uio/uio*/device/resource*'; do
 
-		if list=$(ls "$glob"); then
+		if list=$(ls $glob); then
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]

Reply:
need to ignore this in order to preserve the gloving and word splitting as we are deferring globbing expansion until here on purpose to avoid the arglist limits from being exceeded

Reply:
FWIW https://github.com/koalaman/shellcheck/wiki/SC2086.